pkg/build/api/v1/types.go

```diff
@@ -525,6 +525,11 @@ type BuildConfigSpec struct {
 	// are defined, a new build can only occur as a result of an explicit client build creation.
 	Triggers []BuildTriggerPolicy `json:"triggers"`
 
+	// SchedulePolicy describes how the new build created from this build
+	// configuration will be scheduled for execution.
+	// This is optional, if not specified we default to "parallel".
+	SchedulePolicy *BuildSchedulePolicy `json:"schedulePolicy,omitempty"`
```

Comment:
NewBuildPolicy? StartPolicy? (I like the later, because it describes the purpose of this field... it is a policy of how the builds are started - serially, parallel or latest build only).